src/components/LargeCardMain.jsx

```diff
@@ -19,6 +21,22 @@ export default function LargeCardMain(props) {
       .catch((error) => console.log(error));
   }, [id]);
 
+
+  const addPlantToGarden = () => {
+    const info = {
+      userId: 1,
+      plantId: id
+    }
+
+    axios.post("/api/my_garden/submit", info).then((response) => {
+      console.log(response.data);
+      navigate("/my_garden");
```

Comment:
Do we still need this console log of the data?

Reply:
Removed